docs/source/extras.accelerators.rst

```diff
@@ -0,0 +1,4 @@
+.. automodule:: flytekit.extras.accelerators
```

Comment:
Should this just be `flytekit.extras`?

Reply:
no accelerators is a specific module under extras